palindrome.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <string.h>
+int palindrome(int length, int i, char pal[]){
```

Comment:
this function did return a warning, and it's because it's possible that the input doesn't return a value since there is no `else ` case